AEPNotificationContent/Sources/Controllers/TemplateController.swift

```diff
@@ -49,6 +61,12 @@ class TemplateController: UIViewController {
         fatalError("BaseTemplateController cannot be initialized from storyboard.")
     }
 
+    func activateTapGesture() {
+        if (view.gestureRecognizers?.contains(tapGestureRecognizer)) == nil {
+            view.addGestureRecognizer(tapGestureRecognizer)
+        }
+    }
```

Comment:
this implementation is overly convoluted. could be simplified if we changed `tapGestureRecognizer` to an optional property then doing something like this:
```
guard recognizer != nil else { return }

recognizer = makeRecognizer
view.add(recognizer)
```

Reply:
I see. Yeah this could save some processing. Changed, Let me know how it looks